### DISCUSSION TITLE
DnnImageHandler: why not friendly url?

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4424] [Q&A]
exp: dnndev.me/dnnimage/profilepic/32x32/u-xxx-displayname.jpg
I don't understand why not implementing Friendly URLs for Image Urls?
What's the matter here? Concerned about performance or what?

**Comment by mitchelsellers** [comment]
I would say performance is a first concern, but at the same time, it has not be evaluated in a number of years and might be an option now

**Reply by thienvc** [reply]
> I would say performance is a first concern, but at the same time, it has not be evaluated in a number of years and might be an option now

https://github.com/dnnsoftware/Dnn.Platform/blob/4eaa5e62074731f4d07b5a1f4355d7933d741292/DNN%20Platform/Library/Services/GeneratedImage/DnnImageHandler.cs#L34-L39

I want to customize WhiteList folder, because I miss deploying many websites where `PortalHomeFolder `custom path is not in format `/Portals/xx/`
In your opinion, how should I make it reasonable?
